CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+# Contributing to GraphQL-Flutter
+
+We would love for you to contribute to graphql-flutter and help make it even better than it is
+today! As a contributor, here are the guidelines we would like you to follow:
+
+- [Question or Problem?](#question)
+- [Issues and Bugs](#issue)
+- [Feature Requests](#feature)
+- [Submission Guidelines](#submit)
+- [Coding Rules](#rules)
+- [Commit Message Guidelines](#commit)
+
+## <a name="question"></a> Got a Question or Problem?
+
+Do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [Stack Overflow][stackoverflow] where the questions should be tagged with tag `flutter`.
```

Comment:
tagged with [`graphql` and `flutter`](https://stackoverflow.com/questions/tagged/graphql+flutter)